tce/src/tools/Environment.cc

```diff
@@ -1087,8 +1073,27 @@ std::vector<std::string> Environment::implementationTesterTemplatePaths() {
         string path = string(TCE_SRC_ROOT) + DS + "data" + DS + "hdb";
         paths.push_back(path);
     }
-    string path = string(TCE_INSTALLATION_ROOT) + string(INSTALLATION_DIR) 
-        + "data" + DS + "hdb";
+    string path = instPath() + string(INSTALLATION_DIR) + "data" + DS + "hdb";
     paths.push_back(path);
     return paths;
 }
+
+/**
+ * Returns the installation path for TCE
```

Comment:
Check the text in the Doxygen comment: First a brief single sentence and then an empty line, then a more verbose comment, if needed. Also please describe what the TCE_USER_INSTALL should be set to.